unattended.sh

```diff
@@ -25,15 +27,29 @@ print_step_begin "System analysis"
 
 # Unattended install script only support Red Hat or compatible.
 if [ \! -e /etc/redhat-release ] ; then
-  error_and_exit "This unattended installation script only supports Red Hat compatible distributions. Please check https://documentation.centreon.com/20.10/en/installation/introduction.html for alternative installation methods."
+  error_and_exit "This unattended installation script only supports Red Hat compatible distributions. Please check https://documentation.centreon.com/$CENTREON_MAJOR_VERSION/en/installation/introduction.html for alternative installation methods."
 fi
-rhrelease=`cat /etc/redhat-release`
+rhrelease=$(rpm -E %{rhel})
 case "$rhrelease" in
-  'CentOS Linux release 7.'*)
+  '7')
+    # Good to go.
```

Comment:
this comment does not clarify the intention
Good to go -> Begin CentOs7 script

Reply:
I remove the comment
It was in the previous code to indicate the only case when the rest of the script could be executed
Now there are 2 cases, so indeed il doesn't clarify the intention anymore